AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -577,9 +600,6 @@ - (void)insertSections:(NSIndexSet *)sections withAnimationOptions:(ASDataContro
       NSMutableArray *updatedNodes = [NSMutableArray array];
       NSMutableArray *updatedIndexPaths = [NSMutableArray array];
       [self _populateFromDataSourceWithSectionIndexSet:sections mutableNodes:updatedNodes mutableIndexPaths:updatedIndexPaths];
-      
-      // Measure nodes whose views are loaded before we leave the main thread
-      [self layoutLoadedNodes:updatedNodes ofKind:ASDataControllerRowNodeKind atIndexPaths:updatedIndexPaths];
```

Comment:
Glad you found a cleaner way to integrate this at the layout dispatch level.  This has always bothered me.  Fortunately, in a couple months the work on the new threading model should allow the distinction to be eliminated for loaded nodes.